uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/PublicHoliday.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<table>
+  <base>CRM/HRLeaveAndAbsences</base>
+  <class>PublicHoliday</class>
+  <name>civicrm_hrleaveandabsences_public_holiday</name>
+  <comment>FIXME</comment>
```

Comment:
I also always forget to fix this FIXME! I think we can left it blank and add some description